.changeset/sour-papayas-wink.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/catalog-model': minor
```

Comment:
Technically the json schema is a breaking change but I could not find it being used outside of this package. If you want I can change it to be a major bump.

Reply:
This is an output type since relations only existing on entities read from the catalog. That means that the only breaking change here is to make `target` no longer required. I think we can avoid the breaking change by keeping it marked as `required` as long as it's still in the schema.

Reply:
@Rugvip Updated based on your comments to continue to have target as required